plugins/cd/cdprefs.py

```diff
@@ -30,7 +30,6 @@
 from xl import settings, transcoder
 from xl.nls import gettext as _
 from xlgui.preferences import widgets
-from __builtin__ import str
```

Comment:
Or should stash this change into the original pull request which introduced this change?